legateboost/metrics.py

```diff
@@ -194,7 +206,7 @@ class GammaDevianceMetric(BaseMetric):
     :math:`E = 2[(\\ln{\\frac{p_i}{y_i}} + \\frac{y_i}{p_i} - 1)w_i]`
     """
 
-    def metric(self, y: cn.ndarray, pred: cn.ndarray, w: cn.ndarray) -> float:
+    def metric(self, y: cn.ndarray, pred: cn.ndarray, w: cn.ndarray) -> cn.ndarray:
```

Comment:
hmmm can you help me understand this type hint? The final line of this function is

```python
return float(result)
```

Does `cunumeric` do something to make `float()` called on a `cunumeric.ndarray` also return an `ndarray`, instead of a floating-point scalar?

Reply:
This was a mistake :)